jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SendMessagePane.kt

```diff
@@ -2,8 +2,119 @@
 // SPDX-License-Identifier: Apache-2.0
 package software.aws.toolkits.jetbrains.services.sqs.toolwindow
 
+import com.intellij.icons.AllIcons
+import com.intellij.ide.HelpTooltip
+import com.intellij.ui.components.JBTextArea
+import com.intellij.ui.components.JBTextField
+import com.intellij.ui.components.panels.NonOpaquePanel
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import software.amazon.awssdk.services.sqs.SqsClient
+import software.aws.toolkits.jetbrains.services.sqs.Queue
+import software.aws.toolkits.jetbrains.utils.ApplicationThreadPoolScope
+import software.aws.toolkits.resources.message
+import javax.swing.JButton
+import javax.swing.JLabel
 import javax.swing.JPanel
 
-class SendMessagePane {
+class SendMessagePane(
+    private val client: SqsClient,
+    private val queue: Queue
+) : CoroutineScope by ApplicationThreadPoolScope("SendMessagePane") {
     lateinit var component: JPanel
+    lateinit var inputText: JBTextArea
+    lateinit var fifoComponent: JPanel
```

Comment:
`fifoComponent` does not really explain what it is. Maybe `fifoFields` or `fifoArguments` or something to that effect